tools/roles/pytorch/tasks/pytorch_verify.yml

```diff
@@ -97,7 +97,7 @@
     - name: Set gaudi_image_run_cmd
       ansible.builtin.set_fact:
         gaudi_image_run_cmd: "nerdctl run -it --privileged -e HABANA_VISIBLE_DEVICES=all -e OMPI_MCA_btl_vader_single_copy_mechanism=none --cap-add=sys_nice
-           --net=host --ipc=host -v /opt/omnia/:/workspace/ {{ pytorch_gaudi_image }} python /workspace/pytorch_example.py"
```

Comment:
why are we changing this?

Reply:
it was defect in 1.7.1 just kept it updated. But this is nothing to do with slurm, will revert the changes